cypress/objects/loginPage.js

```diff
@@ -0,0 +1,27 @@
+class LoginPage {
+    visit() {
+        cy.visit('/');
+    }
+
+    getUserFieldSelectorById() {
```

Comment:
Method name could be clearer, because now this method does not return a selector, it returns element which you can click later. I would name this `getUserField`

Reply:
fixed with commit 'f073c5'